qiita_db/artifact.py

```diff
@@ -100,6 +100,57 @@ def types():
             qdb.sql_connection.TRN.add(sql)
             return qdb.sql_connection.TRN.execute_fetchindex()
 
+    @staticmethod
+    def create_type(name, description, can_be_submitted_to_ebi,
+                    can_be_submitted_to_vamps, filepath_types):
+        """Creates a new artifact type in the system
+
+        Parameters
+        ----------
+        name : str
+            The artifact type name
+        description : str
+            The artifact type description
+        can_be_submitted_to_ebi : bool
+            Whether the artifact type can be submitted to EBI or not
+        can_be_submitted_to_vamps : bool
+            Whether the artifact type can be submitted to VAMPS or not
+        filepath_types : list of (str, bool)
+            The list filepath types that the new artifact type supports, and
+            if they're required or not in an artifact instance of this type
+
+        Raises
```

Comment:
This is creating a new artifact type (not a new artifact) and I couldn't think in any other case where a new type is unacceptable.